lib/block-supports/typography.php

```diff
@@ -413,36 +420,36 @@ function gutenberg_get_computed_fluid_typography_value( $args = array() ) {
  * Returns a font-size value based on a given font-size preset.
  * Takes into account fluid typography parameters and attempts to return a css formula depending on available, valid values.
  *
- * @param array $preset                     {
- *     Required. fontSizes preset value as seen in theme.json.
+ * @param array $font_size                     {
+ *     Required. fontSizes represented in the preset format as seen in theme.json.
```

Comment:
Tiny nit: now that the param is `$font_size` what if we reorder the wording here?

```suggestion
 *     Required. A font size as represented in the fontSizes preset format as seen in theme.json.
```

Reply:
Yes! Thanks. 

Just to note the reason behind the changed arg name: now that any font size can be fluidified, "preset" no longer makes sense.